src/components/expansion/ExpansionPanel.svelte

```diff
@@ -136,7 +140,10 @@
   <svelte:fragment slot="body">
     <div class="expansion-panel-body">
       <fieldset>
-        <label for="expansionSet">Expansion Set</label>
+        <label for="expansionSet" class="expansion-set-selector">
+          Expansion Set
+          <a href={`${base}/expansion/rules`} target="_blank" rel="noopener noreferrer">View All Expansion Sets</a>
```

Comment:
Based on the text "View All Expansion Sets", I think the href URL here should be `${base}/expansion/sets`